app/controllers/merchant/orders_controller.rb

```diff
@@ -0,0 +1,48 @@
+class Merchant::OrdersController < Merchant::BaseController
+  def show
+    @orders = find_orders
+  end
+
+  private
+
+  def find_orders
+    @items = Item.where("merchant_id = #{current_user.merchant_id}")
```

Comment:
A long method, but seems to be pulling together a lot of this story. Well done.

-

Reply:
We might want to use @items = Item.where("merchant_id =?", current_user.merchant_id) to make things more secure.